webservice/src/test/java/pl/edu/pjatk/lnpayments/webservice/admin/resource/AdminResourceTest.java

```diff
@@ -54,4 +55,17 @@ void shouldReturnOkForUploadKeys() {
         verify(adminService).uploadKey("email", "2137");
     }
 
+    @Test
+    void shouldReturnOkWhenUserRemoved() {
+        AdminRequest request = new AdminRequest("test@test.pl", "test", "test");
+        AdminDeleteRequest getEmail = new AdminDeleteRequest(request.getEmail());
+
+        ResponseEntity<?> responseCreate = adminResource.addAdmin(request);
+        ResponseEntity<?> responseDelete = adminResource.deleteAdmin(getEmail);
+
+        assertThat(responseCreate.getStatusCode()).isEqualTo(HttpStatus.CREATED);
```

Comment:
Ten test nie sprawdza dodawania admina, więc to jaki status otrzymamy nas nie obchodzi. Powinno też raczej użyć jakiegoś repozytorium do dodania danych, bo serwis zawsze może się, a repozytorium nie (nie jesteśmy za nie odpowiedzialni).